### PR TITLE
Add _set_up_subcommand function

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #405

### CODE REVIEW

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: release_management/create_release.py [hunk 1]
```diff
@@ -62,6 +62,18 @@ def _publish():
     subprocess.run(["cargo", "publish"], check=True)
 
 
+def _set_up_subcommand(subcmd, subparsers, help_string, target_func):
+    """
+    Set up subcommand parsers
+    :param str subcmd: the name of the subcommand
+    :param argparse subparsers: the subparsers variable
+    :param str help_string: the help string
```
[the remark this answers is not in the document]
Comment (bgurney-rh) [reply release_management/create_release.py]: Resolved.

**Comment by mulkieran** [comment]
> This should cover all of the Rust libraries that don't have any extra arguments (except for stratisd_proc_macros; should I convert that?)

Yes. Please. Do stratisd_proc_macros as well.

**Comment by bgurney-rh** [comment]
OK; I updated that yesterday, but forgot to push.

**Comment by bgurney-rh** [comment]
Squashed into a single commit.

**Review by mulkieran — CHANGES_REQUESTED** [review]
The refactoring looks good.

I have two further requests for this PR:

1. Change the name "libblkid" to "libblkid-rs".  "libblkid-rs" is both the repo name and the package name, so it is a better choice. For exactly the same reasons, "libcryptsetup" should be "libcryptsetup-rs". 

2. Remove ``` # pylint: disable=too-many-locals``` from the ```_get_parser``` method. With your refactoring, it's no longer the case.

You can leave both these requested changes in separate commits.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**